fe/fe-core/src/main/java/org/apache/doris/planner/RuntimeFilterGenerator.java

```diff
@@ -321,7 +322,7 @@ private void finalizeRuntimeFilter(RuntimeFilter runtimeFilter) {
      * 2. Only olap scan nodes are supported:
      */
     private void assignRuntimeFilters(ScanNode scanNode) {
-        if (!(scanNode instanceof OlapScanNode)) {
+        if (!(scanNode instanceof OlapScanNode) && !(scanNode instanceof ExternalFileScanNode)) {
```

Comment:
I suppose ExternalFileScanNode is used only when `enable_multi_catalog` is set to true.

Reply:
Yes